Tumusime14/Certificate-Manager---David
# TASK-1-KAN-8 SETTING UP PROJECT

#### PR description
- Project setting up and configuration
- Added necessary dependencies.

#### Ticket / Jira Link
- [KAN-8##](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?selectedIssue=KAN-8)

#### Checklist:


#### How To Test Changes

1. Clone the repository
2. Navigate to the directory of the project
3. Install dependencies using the proper syntax (npm install)
4. Run the project with "npm start"
5. Verify that the project runs and all dependencies are correctly set up


#### Additional Comments
- Some configuration files may require further adjustments for good performance.

####Screenshots
##### Output
[Output]![image](https://github.com/user-attachments/assets/e8856774-cedf-47fd-8e8c-956a15c91a65)
